repose-aggregator/components/filters/header-user-filter/pom.xml

```diff
@@ -48,6 +48,7 @@
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_${scala.MajDotMin}</artifactId>
+            <scope>test</scope>
```

Comment:
I made this comment at the top. Should we remove the scoping since is it already set in the dependency management?